drush/Commands/cgov/CgovUserCommands.php

```diff
@@ -314,6 +319,10 @@ protected function loadAdditionalUser($user) {
       // Update the existing account.
       $this->logger->warning(dt('User !user exists already. Updating...', ['!user' => $name]));
       $account->setpassword($pwd);
+      if (array_key_exists('preferred_admin_langcode', $user)) {
+        $user['preferred_admin_langcode'] = ($user['preferred_admin_langcode'] === 'no-preference') ? '' : $user['preferred_admin_langcode'];
```

Comment:
This should not be needed. 

If the key exists in the users file (`(array_key_exists('preferred_admin_langcode', $user))`), then set it on the account (`$account->set('preferred_admin_langcode', $preferred_admin_langcode);`). 

This way we ensure users in the system match the file we have set. Additionally `$preferred_admin_langcode` matches what you are doing for a new user. Which makes the code above to set `$preferred_admin_langcode` make sense all the way up there.